### DISCUSSION TITLE
Scoop and Apt

### DISCUSSION THREAD

**Discussion by trymeouteh** [dail8859/NotepadNext#618] [Ideas]
Please release this app as a Windows scoop package and release this app in the Debian Apt repository.

I do expect the Debian Apt repository release to take some time and to always be out of date but will be simple to download on Linux.

**Comment by dail8859** [comment]
> Please release this app as a Windows scoop package

I didn't add it and I don't maintain the scoop package but it looks like someone added it: https://github.com/ScoopInstaller/Extras/blob/master/bucket/notepadnext.json

> release this app in the Debian Apt repository

There's already various ways to install it on Linux (offical and unoffical ways). I also don't use Linux so I can't really offer any support here and not sure what kind of effort it would take to maintain an apt package.

**Reply by trymeouteh** [reply]
> not sure what kind of effort it would take to maintain an apt package.

I am not sure either, but the apt packages are not updated often in the official apt repositories and I think that is fine for a text editor.
